### ISSUE TITLE
Enabling HDCD decoding results in empty files

### ISSUE THREAD

**Issue by dwo** [cyanreg/cyanrip#82]
I've been trying to process a well-known HDCD (Green Day's Nimrod from 1997), but using `-H` results in empty output files. I tried ALAC, FLAC, and WAV outputs. 

The log does suspiciously include `HDCD detected: no`: [nimrod..log](https://github.com/cyanreg/cyanrip/files/14893896/nimrod.log)

However, if I rip a WAV without `-H` and then pass it through `ffmpeg -af hdcd`, it detects and handles the HDCD encoding fine:
```
[Parsed_hdcd_0 @ 0x64f3783cbb00] HDCD detected: yes, peak_extend: enabled permanently, max_gain_adj: -4.0 dB, transient_filter: not detected, detectable errors: 0
```

Hope this is enough detail, thank you the excellent tool. 🙇

**Comment by cyanreg** [comment]
FFmpeg changed something with filtering. Fixed in be23815c7c24725f7cb28035abf3208525181c97
Thanks for the report

**Comment by dwo** [comment]
You're welcome, and thank you again. 🙇
